rules/format-new-lines/rule.md

```diff
@@ -56,6 +56,32 @@ https://youtu.be/dfE_Y8fy_wo?si=NEcQLAPafAWKa7m5
 Figure: Good Example - The URL being on a fresh line makes it much easier to read.
 :::
 
+This is also recommended when sending PBIs for better readability.
+
+::: greybox
+Hey Adam, 
+
+I have 2 PBIs on my next to-do in the coming sprint: [Product Backlog Item 88994](https://dev.azure.com/ssw/SSW.Induction/_workitems/edit/88994): ⚡Performance | Create a new App Service plan and [Product Backlog Item 88823](https://dev.azure.com/ssw/SSW.Induction/_workitems/edit/88823): 🚗 Azure | Create a new App Service Plan in West US for SL production resource group.
```

Comment:
I think we can remove the link - and just write the PBI as is :)

Reply:
> I think we can remove the link - and just write the PBI as is :)

✅ Done